src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -3143,8 +3143,7 @@ SupportedLateralQuery(Query *parentQuery, Query *lateralQuery)
 	{
 		OpExpr *operatorExpression = NULL;
 		List *argumentList = NIL;
-		char *operatorName = NULL;
-		int equalsOperator = 0;
+		bool equalsOperator = false;
```

Comment:
it contains the same information, unfortunately there is not much naming alternatives here